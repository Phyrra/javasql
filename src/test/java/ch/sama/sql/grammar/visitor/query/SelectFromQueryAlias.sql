select alias.*
from (
  select * from table
) as alias