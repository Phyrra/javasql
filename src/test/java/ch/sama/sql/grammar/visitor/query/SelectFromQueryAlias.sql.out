SELECT [alias].*
FROM (
SELECT *
FROM [table]
) AS [alias]